src/model/schema/ThunderstoreSchema.ts

```diff
@@ -0,0 +1,131 @@
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
+import schema from "../../../schema.json";
+import schemaSchema from "../../../schemaSchema.json";
+import { GameSelectionDisplayMode } from "../game/GameSelectionDisplayMode";
+import R2Error from "../errors/R2Error";
+
+// Disabled for now, will become relevant when we migrate away from JSON modules.
+export interface Schema {
```

Comment:
IMO the biggest issue in this PR is that we have to redefine the interfaces manually, and then having two sources of truth. Based on ajv's docs it should be possible to use schemaSchema as the single source of truth, although I'm not sure if it can be used with JSON modules. But it might work with very minimal manual changes (although that might pose issues when we start reading the schema from online?)

So IMO it would be worth investing some time to see if we can do without these interfaces, either with ajv or some other library.

Reply:
These are not necessary while we use JSON modules. I was playing around with the idea of using these types as a facade in order to make the transition to a dynamic schema easier, but that's not especially relevant right now.

I'm working through these comments top-to-bottom so you might've commented on this later, but AJV is already the source of truth for the schema JSON.